### DISCUSSION TITLE
vs2019 cake can not find task runner explorer

### DISCUSSION THREAD

**Discussion by jingliancui** [cake-build/cake#3323] [Extension Q&A]
After extension for vs2019 by this helper link,
https://cakebuild.net/docs/integrations/editors/visualstudio/task-runner

I can't not find task runner explorer anywhere in vs .
My project is a xamarin.android binding project.

**Comment by nils-a** [comment]
@jingliancui for me, the *Task Runner Explorer* is available via View --> Other Windows --> Task Runner Explorer.

![image](https://user-images.githubusercontent.com/349188/116591831-955e5600-a91f-11eb-8d77-600299ae11b2.png)

Keep in mind, that the Cake file needs to be named `build.cake` and has to be placed next to the solution, for the Task Runner Explorer to pick it up.

**Reply by jingliancui** [reply]
@nils-a Hi nils, it's useless.
![无标题](https://user-images.githubusercontent.com/16309962/116637452-9383a980-a996-11eb-99e6-8c1937f409ed.png)

**Reply by nils-a** [reply]
@jingliancui Which edition and version of VS are you using and which version of Cake for Visual Studio is installed?
